doc/source/Interface/ControlsAndShortcuts/index.md

```diff
@@ -0,0 +1,171 @@
+# Controls and Shortcuts #
+
+The following is a list of input device controls and shortcuts for manipulating Gaffer's _Node Graph_, _Viewer_, and _Script Editor_ tabs.
+
+
+## _Node Graph_ Tab ##
+
+> Note :
+> For the following controls and shortcuts, the cursor must hover inside the _Node Graph_ tab.
+
+### Node Creation ###
+
+```eval_rst
+===================================== =============================================
+Action			                      Shortcut or Gesture
+===================================== =============================================
+Show node menu                        Right-click
```

Comment:
I love the keyboard style. Is there an equivalent visual mouse button clicks?

Reply:
Could be done with .svgs (dynamic options require HTML or Sphinx modification, which John and I don't want). My only concern is for tablet users, but I suppose that's minor.